### DISCUSSION TITLE
sub-process patching when file depends on #! for python executable

### DISCUSSION THREAD

**Discussion by karthiknadig** [microsoft/debugpy#1184] [Enhancements]
The command line that it generates for subprocesses has `./manage.py` as the first entry in the list that it passes to `subprocess.Popen()` - try inserting `sys.executable` there.

_Originally posted by @int19h in https://github.com/microsoft/debugpy/issues/265#issuecomment-634405386_

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
